inst/help/auditbayesianplanning.md

```diff
@@ -84,6 +84,8 @@ When both risk assessments are set to High (100%) the audit risk model is not us
 #### Expected errors in sample
 The expected errors are the tolerable errors that can be found in the sample while still achieving the specified sampling objectives. A sample size is calculated so that, when the number of expected errors is found in the sample, the desired confidence is retained.
 
+*Note:* It is advised to set this value conservatively to minimize the probability of the observed errors exceeding the expected errors, which would imply that insufficient work has been done.
```

Comment:
I think at the moment it's still required to also do the _nl translations yourself for help files

Reply:
Yes, this is not yet in WebLate.